### PR TITLE
Download an Asset updates

### PR DESCRIPTION
**Pull request by wwarner-salesforce**
https://docs.google.com/spreadsheets/d/1bYSveWDm4nzaPPXqCfv4T5E2DKhVV4NQ9w5Wl7GVBHM/ line 10

Spreadsheet note says "Drop-down needs to be fixed + need to list all file types"

Drop-down needs to be fixed: Is this the formatting problem, which I've hopefully corrected here, or something different?

need to list all file types:

I tried to make the Download an Asset page match the drop-down on qax by adding Connector, Template, and Example with the behavior I observed from the first few results. I think "REST APIs" is covered by RAML and OAS. Choosing SOAP APIs on the drop-down only showed two results; the first had no Download link and instead had a View WSDL link that led to an error page, and the second had a Download link that saved an empty file named test-api-1.1.0-wsdl.docx . Choosing HTTP APIs and API Spec Fragments on the drop-down showed no results.

drop-down asset types on qax: Connectors, Templates, Examples, REST APIs, SOAP APIs, HTTP APIs, API Spec Fragments, Custom
All Types Filters on Find Asset Information page: Connectors, Templates, Examples, REST APIs, SOAP APIs, HTTP APIs, RAML Fragments, Custom
Content Types on About Exchange page: Connectors, REST files, SOAP files, HTTP APIs, Examples, Templates, Custom
file types on Download an Asset page: RAML, OAS, WSDL, Custom with a file, Custom without a file

Was this the right approach to list all file types, or do I need to approach this differently?

### CODE REVIEW

File: modules/ROOT/pages/to-download-an-asset.adoc [hunk 1]
```diff
@@ -6,20 +6,25 @@ endif::[]
...
+After creating a REST asset, refresh the page before clicking Download.
+
 Click the Download button in the task bar. image:ex2-icon-download.png[Download button in the taskbar]
 
 The results vary by file type:
 
-* RAML: Downloads a zip file of the RAML file.
+* RAML: Click the down arrow.
++
+image:ex2-download-from-raml.png[Download menu for RAML with RAML and OAS and Mule 4 connector and Mule 3 connector choices]
```
Comment (george-hoffer) [inline modules/ROOT/pages/to-download-an-asset.adoc]: Should be image::ex2-download-from-raml.png[]

File: modules/ROOT/pages/to-download-an-asset.adoc [hunk 2]
```diff
@@ -6,20 +6,25 @@ endif::[]
...
++
+If you choose As RAML the original RAML file downloads as a zip file. If you choose As OAS, Exchange converts the file and it downloads as a zip file containing JSON. If you choose As Mule 4 connector, it downloads as a jar file you can add to a Studio 7 Mule project. If you choose As Mule 3 connector, it downloads as a zip file you can add to a Studio 6 Mule project.
```
Comment (george-hoffer) [inline modules/ROOT/pages/to-download-an-asset.adoc]: Missing a comma: "If you choose As RAML, "

Comment (george-hoffer) [reply modules/ROOT/pages/to-download-an-asset.adoc]: Also: "If you choose As OAS, Exchange converts the file and it downloads as a zip file containing JSON"
Could be better as:
"If you choose As OAS, Exchange downloads the API specification as a zip file containing a JSON file for the specification."

Comment (george-hoffer) [reply modules/ROOT/pages/to-download-an-asset.adoc]: Need to stay parallel -- we can't assume the original is either a RAML or OAS unless you have specific sections for each.

File: modules/ROOT/pages/to-download-an-asset.adoc [hunk 3]
```diff
@@ -6,20 +6,25 @@ endif::[]
...
++
 * OAS: Click the down arrow.
 +
-image:ex2-icon-download-with-arrow.png[Download button for OAS with OAS and RAML choices]
+image:ex2-download-from-oas.png[Download menu for OAS with RAML and OAS and Mule 4 connector and Mule 3 connector choices]
```
Comment (george-hoffer) [inline modules/ROOT/pages/to-download-an-asset.adoc]: Image tag should be:  image::ex2-download-from-oas[]

File: modules/ROOT/pages/to-download-an-asset.adoc [hunk 4]
```diff
@@ -6,20 +6,25 @@ endif::[]
...
 +
-Choose the format As OAS or As RAML. If the original is in the
-alternate format, Exchange converts the file to your choice. OAS downloads in the original format
-of either JSON or YAML. A RAML downloads as a zip file.
+If you choose As RAML, Exchange converts the file and it downloads as a zip file containing a RAML file. If you choose As OAS it downloads as a zip file containing a file in the original format of either JSON or YAML. If you choose As Mule 4 connector, it downloads as a jar file you can add to a Studio 7 Mule project. If you choose As Mule 3 connector, it downloads as a zip file you can add to a Studio 6 Mule project.
```
Comment (george-hoffer) [inline modules/ROOT/pages/to-download-an-asset.adoc]: Same problem as above. Please make the sections parallel and better not to mention converting it has no user context. Also "it downloads" sounds strange versus "downloads it".

File: modules/ROOT/pages/to-download-an-asset.adoc [hunk 5]
```diff
@@ -1,21 +1,32 @@
-= To Download an Asset
+= Download an Asset
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
+:imagesdir: ../assets/images/
 
 Download lets you download a copy of the asset, or in the case of a Custom asset, the optional file attached to the asset when it was created.
 
+After creating a REST asset, refresh the page before clicking Download.
```
Comment (nicolasgnr) [inline modules/ROOT/pages/to-download-an-asset.adoc]: We should delete this and explain that the Mule3 and Mule4 connectors are generated in background. A refresh does not guarantee that the corresponding download button will appear.

Comment (wwarner-salesforce) [reply modules/ROOT/pages/to-download-an-asset.adoc]: Good point. Fixing now.

**Comment by wwarner-salesforce** [comment]
@george-hoffer I think this one is ready for review.

**Review by george-hoffer — APPROVED** [review]

**Pull request merged**